easycv/datasets/loader/sampler.py

```diff
@@ -218,7 +220,6 @@ def generate_new_list(self):
     def set_uniform_indices(self, labels, num_classes):
         self.unif_sampling_flag = True
         assert self.shuffle, 'Using uniform sampling, the indices must be shuffled.'
-        np.random.seed(self.epoch)
```

Comment:
why remove？

Reply:
Seed should be managed uniformly and should not be abused